bot/cogs/birthday/birthday_commands.py

```diff
@@ -63,12 +95,45 @@ async def birthday_admin(self, interaction: Interaction):
 
     @birthday_admin.subcommand(name='config', description='Configure birthday settings')
     async def birthday_config(self, interaction: Interaction,
-                              channel: TextChannel = SlashOption(name='channel',
-                                                                 description='Channel to post birthday messages in.')):
-        success = birthday_helper.update_settings(interaction.guild_id, channel.id)
-        if not success:
-            return await interaction.send(f'An error occurred when updating birthday settings.', ephemeral=True)
+                                birthday_channel: TextChannel = SlashOption(name='birthday_channel',
+                                                                description='Channel to post birthday messages in.', required=False),
+                                baby_month_milestone_channel: TextChannel = SlashOption(name='baby_month_milestone_channel', 
+                                                                description='Channel to post baby month milestone messages in.', required=False)):
+        if birthday_channel is not None:
+            success = birthday_helper.update_birthday_channel_settings(interaction.guild_id, birthday_channel.id)
+            if not success:
+                return await interaction.send(f'An error occurred when updating birthday channel settings.')
+            
+        if baby_month_milestone_channel is not None:
+            success = birthday_helper.update_baby_month_channel_settings(interaction.guild_id, baby_month_milestone_channel.id)
+            if not success:
+                return await interaction.send(f'An error occurred when updating baby milestone channel settings.', ephemeral=True)
         await interaction.send('Successfully updated birthday settings!', ephemeral=True)
+    
```

Comment:
Added the birthday add/remove commands for admins to be able to add/remove birthdays for any user